Kronos-Shifts-Connector/Deployment/azuredeploy.json

```diff
@@ -729,7 +772,7 @@
                 "attributes": {
                     "enabled": true
                 },
-                "value": "[variables('defaultKronosUserName')]"
+                "value": "[variables('defaultKronosUsername')]"
```

Comment:
Why is this a variable, and not a parameter?

Reply:
This has been resolved, I have made these values into parameters